discovery_engine_core/web-api/Dockerfile

```diff
@@ -5,6 +5,5 @@ WORKDIR /service
 COPY ./ ./
 
 RUN chmod +x ./web-api
-VOLUME /service/assets/data.json
```

Comment:
why remove the volume? you don't have to use it, but it gives additional possibility to change data without rebuilding the container

Reply:
In fact it was pretty tricky to use the volume in our kubernetis setup - or maybe I and Antonie were too stupid to do it 
Now I am not sure how this behaves when you have both Volume and the file but I can test it.